recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -169,14 +185,30 @@ CONFIG_DATE=y
 # CONFIG_FEATURE_DATE_ISOFMT is not set
 # CONFIG_FEATURE_DATE_NANO is not set
 # CONFIG_FEATURE_DATE_COMPAT is not set
+# CONFIG_DD is not set
```

Comment:
You are disabling dd, which was disabled in the old recipe.  Is that on purpose?